vllm/worker/worker.py

```diff
@@ -229,6 +323,49 @@ def remove_lora(self, lora_id: int) -> bool:
     def list_loras(self) -> Set[int]:
         return self.model_runner.list_loras()
 
+    def should_execute(self, is_prompt: bool) -> bool:
+        return self.is_mixed_worker() or (
+            self.is_prompt_worker() and is_prompt) or (self.is_token_worker()
+                                                       and not is_prompt)
+
+    def set_gpucache(self):
+        from vllm.worker.comm_utils import HEAD_TYPES
+        num_layers = self.model_config.get_num_layers(self.parallel_config)
+        for layer_id in range(num_layers):
+            for head_type in HEAD_TYPES:
+                self.gpu_cache[layer_id][head_type][:] = self.rank * (
+                    num_layers *
+                    len(HEAD_TYPES)) + layer_id * len(HEAD_TYPES) + head_type
+        torch.cuda.synchronize()
+
+    def send_recv_kvcache_all(self):
```

Comment:
is this method only use in testing? I can not find any other references. If so, can you add some comments?